### ISSUE TITLE
Overhaul templates/shortcodes/gallery.html and screenshotgallery.html to include alt= and title= tags

### ISSUE THREAD

**Issue by 1peter10** [1peter10/minimola#3]
Documentation for this should be at or around https://www.getzola.org/documentation/content/shortcodes/
